src/Library/demos/Breakpoints/main.blp

```diff
@@ -0,0 +1,57 @@
+using Gtk 4.0;
+using Adw 1;
+
+Gtk.Window {
+  width-request: 360;
+  height-request: 200;
+  default-width: 640;
+  default-height: 480;
+  title: _("Breakpoints");
+
+  Box {
+    orientation: vertical;
+    spacing: 12;
+    valign: center;
+
+    Image image {
+      icon-size: large;
+      icon-name: "face-devilish-symbolic";
+    }
+
+    Adw.BreakpointBin breakpoint_bin {
+      width-request: 200;
+      height-request: 50;
+
+      Adw.Breakpoint breakpoint {
+      condition ("max-width: 500sp")
+        setters {
+          breakpoint_bin.child: narrow;
+        }
+      }
+
+      child: wide;
+    }
+
+    LinkButton {
+      margin-top: 24;
+      label: "Breakpoint";
+      uri: "https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.Breakpoint.html";
+    }
+
+    LinkButton {
+      label: "Breakpoint Bin";
+      uri: "https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.BreakpointBin.html";
+    }
+  }
+}
+
+
+Label narrow {
+  label: _("Narrow");
+  styles ["title-4"]
+}
+
+Label wide {
+  label: _("Wide");
+   styles ["title-1"]
+}
```

Comment:
I think this will make clearer what the setter does.

```suggestion
Label label_narrow {
  label: _("Narrow");
  styles ["title-4"]
}

Label label_wide {
  label: _("Wide");
   styles ["title-1"]
}
```